### PR TITLE
Add storage handler plugins.

### PR DESCRIPTION
**Pull request by mbarnes**
I took some liberties with the module names so they read better in a configuration file.

Apart from that and some `import` adjustments, there's no changes from the MVP code.

For example:

```
    "storage-handlers": [
        {
            "name": "commissaire.storage.etcd",
            ...
        }
    ]
```

### CODE REVIEW

File: src/commissaire/storage/kubernetes.py [hunk 1]
```diff
@@ -0,0 +1,560 @@
+# Copyright (C) 2016  Red Hat, Inc
```
Comment (ashcrow) [inline src/commissaire/storage/kubernetes.py]: I don't think we should bring in the kubernetes storage handler. At least not yet. I'd like us to just focus on the etcd backend and pull in others as the need arises.

**Review by ashcrow — CHANGES_REQUESTED** [review]
Over all makes sense with one area I'd like to drop for now.

**Comment by mbarnes** [comment]
:arrow_up: Kube dropped.

**Review by ashcrow — APPROVED** [review]

**Comment by ashcrow** [comment]
:+1:

**Pull request merged**